eland/tests/dataframe/test_repr_pytest.py

```diff
@@ -267,3 +269,27 @@ def test_empty_dataframe_repr_html(self):
         pd.set_option("display.show_dimensions", show_dimensions)
 
         assert ed_ecom_rh == pd_ecom_rh
+
+    def test_dataframe_repr_pd_get_option_none(self):
+        show_dimensions = pd.get_option("display.show_dimensions")
+        show_rows = pd.get_option("display.max_rows")
+
+        pd.set_option("display.show_dimensions", False)
```

Comment:
Since tests can technically fail we should wrap everything in a try-finally:

```python
show_dimensions = pd.get_option("display.show_dimensions")
try:
    pd.set_option("display.show_dimensions", False)

    ... (rest of the test)

finally:
    pd.set_option("display.show_dimensions", show_dimensions)
```

Reply:
Is it a good practice to ignore?
```python
except Exception as e:
    print(e)
```

Reply:
since we're not adding an `except` clause the exception will automatically bubble up.